src/components/course/data/hooks.js

```diff
@@ -305,6 +305,17 @@ export function useCourseEnrollmentUrl({
   );
 
   return enrollmentUrl;
-}
+};
+
```

Comment:
The addition of prop types here will throw an error if a similar bug to this happens again, right?

Reply:
actually I had expected it to work but now that I tried it does not look like proptypes works for hook functions. I am going to leave it here but think there is some investigation to be done on whether it can be made to work (if not maybe just remove the proptypes on both functions here)

Reply:
this seems to hint that prop types does not work for anything but components, which makes me sad :( 

https://stackoverflow.com/questions/54296313/proptypes-for-custom-react-hooks